public/js/components/SourceTabs.css

```diff
@@ -74,6 +74,11 @@
   transition: transform 0.5s ease;
 }
 
+.collapse-button-left:hover,
+.collapse-button-right:hover {
+  cursor: pointer;
+}
+
```

Comment:
None of our current expand/collapse buttons have a pointer, nor does the Chrome devtools do this. I know we've added the pointer icon to a few other places, but we should decide what we want to be consistent. We traditionally didn't use this before (and I kind of like how Chrome feels without it).